docs/client.md

```diff
@@ -308,7 +308,7 @@ client.search('o=example', opts, (err, res) => {
     console.log('searchRequest: ', searchRequest.messageId);
   });
   res.on('searchEntry', (entry) => {
-    console.log('entry: ' + JSON.stringify(entry.object));
+    console.log('entry: ' + JSON.stringify(entry.pojo));
```

Comment:
This looks good according to the example. It's just trying to print out a JSON representation of the full search entry object. If the example were to be reworked to show how to access the attributes, that would be `entry.attributes`.

Reply:
Good to know! I'll try it out in my project.